connectors/connectorLibs/ContentScript.js

```diff
@@ -361,11 +361,26 @@ export default class ContentScript {
   /**
    * Send log message to the launcher
    *
+   * @param {string} : the log level
    * @param {string} : the log message
-   * @todo Use cozy-logger to add logging level and other features
    */
-  log(message) {
-    this.bridge.emit('log', message)
+  log(level, message) {
```

Comment:
We will need to port this to cozy-ccc-libs

Reply:
PR done